### PR TITLE
docs(developer): add instruction how to run aio locally

### PR DESCRIPTION
**Pull request by pmvald**
This test is part of CI and is needed in addition to tests in packages.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: docs/BAZEL.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ new as of May 2017 and not very stable yet.
 - Test package in node: `yarn test packages/core/test:test`
 - Test package in karma: `yarn test packages/core/test:test_web`
 - Test all packages: `yarn test packages/...`
+- Test aganist angular.io app locally: `yarn test //aio/... --config=aio_local_deps`
```
Comment (devversion) [inline docs/BAZEL.md:54]: angular.io has its own readme and helper scripts, but I'm in favor of adding this for completeness:

https://github.com/angular/angular/tree/main/aio

**Review by devversion — APPROVED** [review]

File: docs/BAZEL.md (hunk 1 above)
Comment (josephperrott) [inline docs/BAZEL.md:54]:
```suggestion
- Test angular.io app locally: `yarn test //aio/... --config=aio_local_deps`
```

Comment (pmvald) [reply docs/BAZEL.md]: Done!

Comment (pmvald) [reply docs/BAZEL.md]: Noted

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 273d8e93adde9c143d3fa767714757479d907757.

**Pull request closed without merging**